feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -364,19 +360,4 @@ func aftCheck(t testing.TB, dut *ondatra.DUTDevice, prefix string, expectedNH []
 	if got := len(aftNHG.NextHop); got < 1 && aftNHG.BackupNextHopGroup == nil {
 		t.Fatalf("Prefix %s reachability didn't switch to backup path", prefix)
 	}
-	if len(aftNHG.NextHop) != 0 {
```

Comment:
Is there a reason to remove this code entirely, shouldn't it pass with other cases?

Reply:
Because the entire block of codes says: `for each streamed next hop, it has to be in the expectedNH`, **not** `for each IP in the expectedNHs, it has to be in the streamed next hops`. Basically the code blocks is defining that only the viable NHs can be streamed in the AFT. (which is not a conclusion yet, see the PR description).

Also because the way the code is structured is kinda of confusing. So, I just removed it, instead of refactor it into `for each streamed next hop, it has to be in the expectedNH`.

I added a TODO here same as above.